app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = current_user.comments.new(post_id: @post.id, user_id: current_user.id, Text: comment_params)
+    @comment.post_id = @post.id
+    @comment.update_comment_counter
+    if @comment.save
+      redirect_to user_post_path(current_user, @post.id)
+    else
+      flash.now[:error] = 'Error: Comment could not be saved. Please try again.'
+    end
+  end
```

Comment:
- [ ] I'm not able to create a comment because of the following error. Please ensure the user is able to add comments to the posts.

<img width="907" alt="image" src="https://user-images.githubusercontent.com/77190203/185743196-f9984e18-b8c5-4304-a2f3-31727b06090e.png">